### ISSUE TITLE
View/Edit Member page expires the session 

### ISSUE THREAD

**Issue by carter2422** [restrictcontentpro/restrict-content-pro#488]
As of 2.5.x anytime I `View/Edit Member` on any member with an existing subscription it immediately expires the session and prevents me from ever viewing the member: http://cl.ly/g22M

Initially @pippinsplugins and I thought this was an environment issue, but it seems to be codebase related. As @pippinsplugins noted in a Slack chat, it seems to be breaking while setting up the `RCP_Member` object. 

Everything works fine in 2.4.11.

Webserver config:
```
PHP Version:              5.6.99-hhvm
MySQL Version:            5.6.19
Webserver Info:           nginx/1.9.3
```

**Comment by michaelbeil** [comment]
We are running some tests locally with HgV (HHVM) to confirm the issue.

**Comment by carter2422** [comment]
Thanks Michael!

-- 
Jonathan Williamson

On May 3, 2016 at 1:01:25 PM, Michael Beil (notifications@github.com) wrote:

We are running some tests locally with HgV (HHVM) to confirm the issue.

**Comment by michaelbeil** [comment]
@carter2422 is there anything else specific to your tests that we should be aware of? Caching, browser, other plugins installed, is this on a test server, etc.?

**Comment by pippinsplugins** [comment]
@carter2422 Quick test: can you try it with everything _after_ `new RCP_Member` commented out?

**Comment by carter2422** [comment]
@pippinsplugins already tried that. Still failed. 

@michaelbeil here's the output of the System Config from EDD: http://www.pasteall.org/69003

**Comment by michaelbeil** [comment]
Where are you seeing this error? On a staging site?

**Comment by carter2422** [comment]
@michaelbeil I've replicated it both our [staging site](https://cgcookielab.com) and our [live site](https://cgcookie.com). Note, I've since reverted the live site back to 2.4.11 due to this problem.

**Comment by pippinsplugins** [comment]
@carter2422 Can I get full staging site access? That'll be easiest way to track it down.

**Comment by carter2422** [comment]
@pippinsplugins any luck tracking this down with the staging site access I gave you?

**Comment by pippinsplugins** [comment]
@carter2422 Will try again shortly.

**Comment by pippinsplugins** [comment]
@carter2422 https://cgcookielab.com/wp-admin/ seems to be working fine for me. Is it running latest?

**Comment by pippinsplugins** [comment]
Nvm, it appears to be. I can't make the problem happen.

https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=288798
https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=145710
https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=167349
https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=74003
https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=283582

Those all load fine for me.

**Comment by pippinsplugins** [comment]
@carter2422 Can you give me a user ID that fails for you?

**Comment by carter2422** [comment]
Here's one @pippinsplugins https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=2323

**Comment by carter2422** [comment]
Also @pippinsplugins, of those you listed this one fails for me: https://cgcookielab.com/wp-admin/admin.php?page=rcp-members&edit_member=145710

**Comment by carter2422** [comment]
Any luck on this @pippinsplugins?

**Comment by pippinsplugins** [comment]
@carter2422 that one fails for me. Digging!

**Comment by pippinsplugins** [comment]
@carter2422 can you re-enable the plugin editor for me? https://cloudup.com/cBhYlZ9RSDv

**Comment by pippinsplugins** [comment]
We managed to resolve this miraculously by busting a cache (I think). Will re-open if it comes back.

**Comment by carter2422** [comment]
Sadly it's back soon as we re-deployed the identical codebase. This may be a stack issue but unsure, trying to determine one way or the other.

**Comment by pippinsplugins** [comment]
Can you look directly into the payment database and tell me how many payments there are?

**Comment by carter2422** [comment]
Sure. There's `88,134` rows in `rcp_payments`

**Comment by pippinsplugins** [comment]
This is happening on another customer's site now: https://secure.helpscout.net/conversation/238734192/40096/?folderId=841782

Removing `rcp_print_user_payments_formatted( $member->ID )` from the member details screen resolves it so something in that is killing it.

**Comment by pippinsplugins** [comment]
FOUND IT!

`rcp_print_user_payments_formatted()` is missing an `ob_start()`.

**Comment by carter2422** [comment]
OOOOOOO!

**Comment by pippinsplugins** [comment]
Fixed in e84a256c540a5011d31f8b649869327f073d3292
